app/models/label/LabelValidationTable.scala

```diff
@@ -209,6 +236,38 @@ object LabelValidationTable {
       .update((countsWithoutOldVal._1, countsWithoutOldVal._2, countsWithoutOldVal._3, labelCorrect))
   }
 
+  /**
+   * Updates the validation counts and correctness columns in the label table given a deleted validation.
+   *
+   * @param label  label with the validation we want to delete.
+   */
+  def updateValidationCountsDelete(label: LabelValidation): Int = db.withSession { implicit session =>
```

Comment:
This should definitely just be added to the functionality of `updateValidationCounts()`! You should just make it so that the `newValidationResult` parameter can be an option type, and update that function accordingly. Should be pretty simple I think!